spec/lib/champaign_queue/clients/direct_spec.rb

```diff
@@ -2,12 +2,7 @@
 
 describe ChampaignQueue::Clients::Direct do
   before do
-    ENV['AK_PROCESSOR_URL'] = "http://example.com/message"
-  end
-
-  after do
-    # If this sticks around, it breaks some other tests in other files.
-    ENV['AK_PROCESSOR_URL'] = nil
+    Settings.ak_processor_url = "http://example.com/message"
```

Comment:
We've baked an automatic reset of `Settings` into every test run, meaning that you can modify `Settings` to your heart's content and it will be back to the default `test` setting after your spec finishes.